doc/cephfs/troubleshooting.rst

```diff
@@ -244,24 +244,30 @@ the developers!
 
 Slow requests (MDS)
 -------------------
-You can list current operations via the admin socket by running::
+List current operations via the admin socket by running the following command
+from the MDS host:
 
-  ceph daemon mds.<name> dump_ops_in_flight
+.. prompt:: bash #
+
+   ceph daemon mds.<name> dump_ops_in_flight
 
-from the MDS host. Identify the stuck commands and examine why they are stuck.
+Identify the stuck commands and examine why they are stuck.
 Usually the last "event" will have been an attempt to gather locks, or sending
-the operation off to the MDS log. If it is waiting on the OSDs, fix them. If
```

Comment:
@vshankar: I have a couple of questions about this paragraph.

1. What is "it" in the sentence "It it is waiting on the OSDs, fix them." Is "it" the MDS?
2. What does "fix them" mean in the sentence "If it is waiting on the OSDs, fix them."? Does this mean that the reader should fix the OSDs? What does it mean to "fix" an OSD?

Reply:
I think the antecedent is the MDS but agree the wording could be better while we’re in there.  
In Zac’s defense this is existing text someone else wrote.  Like the “funky” I found recently.  

Agree that an admonition to fix OSDs isn’t helpful.  
first thing I would look for is to ensure that the metadata pool uses a CRUSH rule constrained to an SSD device class, ideally enterprise class NVMe.  I’ve seen these more than once spanning spinners which drags the MDS down.  Next would be to ensure there are enough of those OSDs and that they have a PG ratio of at least 150.